helm/cluster-vsphere/templates/cleanup-helmreleases-hook-job.yaml

```diff
@@ -44,14 +44,10 @@ metadata:
 rules:
   - apiGroups: ["helm.toolkit.fluxcd.io"]
     resources: ["helmreleases"]
```

Comment:
we use generic for loop in the deletion code so it doesn't make sense to list resources explicitly here. less secure but more extendable.